src/app/dashboard/executionsgrid/executionsgrid.component.ts

```diff
@@ -119,11 +120,13 @@ export class ExecutionsgridComponent implements AfterViewInit, OnDestroy {
     ) as Observable<MoreResults<DatasetOverview>>;
 
     this.overviewSubscription = polledOverviewData.subscribe(
-      ({ results, more }) => {
+      ({ results, more, maxResultCountReached }) => {
         this.hasMore = more;
         this.dsOverview = results;
         this.isLoading = false;
         this.isLoadingMore = false;
+        this.maxResultsReached = !!maxResultCountReached;
+        console.error('maxResultCountReached = ' + maxResultCountReached);
```

Comment:
I think this line is your debugging line that should disappear?

Reply:
Hmm... I keep doing that.  Fixed.